ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/loader/JsonTestLoader.java

```diff
@@ -146,10 +145,8 @@ private static void throwOnDuplicateNames(final List<? extends Test> testCases)
         .stream()
         .filter(e -> e.getValue().size() > 1)
         .map(e -> "test name: '" + e.getKey()
-            + "' found in files: " + e.getValue().stream()
-            .map(Test::getTestLocation)
-            .map(TestLocation::toString)
-            .collect(Collectors.joining(",")))
```

Comment:
This was pointless as `getTestLocation` finds the first test in the file with the supplied name - so all get the same location.